packages/inflection-tables/lib/lang/latin/latin-language-dataset.js

```diff
@@ -71,6 +71,10 @@ export default class LatinLanguageDataset extends LanguageDataset {
     }
   }
 
+  static get language () {
+    return Language.LATIN
+  }
+
   static get languageID () {
```

Comment:
do we need languageID getter now?

Reply:
Same as above: I'm not sure, but maybe it's safer to leave as is for now?